src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
```

Comment:
Slightly confused here - https://github.com/dotnet/roslyn-analyzers/blob/release/6.0.1xx/src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md file in `release/6.0.1xx` already has bunch of rules. Did you pull in the latest changes from `release/6.0.1xx` branch?

Reply:
I hadn't. Fixed now